server/entity/MediaRequest.ts

```diff
@@ -444,15 +444,10 @@ export class MediaRequest {
                 label: 'Media Request',
               }
             );
-            const userRepository = getRepository(User);
-            const admin = await userRepository.findOneOrFail({
-              select: ['id', 'plexToken'],
-              order: { id: 'ASC' },
-            });
+
             notificationManager.sendNotification(Notification.MEDIA_FAILED, {
               subject: movie.title,
-              message: 'Movie failed to add to Radarr',
-              notifyUser: admin,
+              message: movie.overview,
```

Comment:
Do we really need the overview on the failed notification? What was wrong with how it was before?

Reply:
The notifications themselves already say that the the request failed, so I feel the original message here is unnecessary.  We don't have to include the plot/overview in the actual notification, but it doesn't hurt, and it would then match with the other media notifications.  I personally like having the plot, since it's in every other notification type, and OCD... 😅

Reply:
TBH though, setting the subject/message here at all is kind of unnecessary for media notifications, since we also pass the media object in the payload.

Reply:
Perhaps. I originally tried to abstract the need to pass along specific objects to display any notification flexibly. I ended up not going that way, so this is a legacy decision. We can change it in the future.